### DISCUSSION TITLE
What is DENOM in the aggregated scores output?

### DISCUSSION THREAD

**Discussion by scienception** [PGScatalog/pgsc_calc#142] [Q&A] [answered]
not sure what this column is for, also looked for it in the documentation but didn't find it :/

**Comment by smlmbrt** [comment, marked as answer]
That's the number of variants with non-missing genotypes, and used to calculate the `[scorename]_AVG` when the # of samples is lower than 50 (to avoid imputing allele frequencies on sample samplesets). This will be on the outputs page of the new documentation in a couple of days. We recommend using the PGS SUM though.

**Reply by scienception** [reply]
Thank you! If the number of samples > 50, then how is [scorename]_AVG computed?

**Comment by Djeane89** [comment]
Hi, is there a way to know how this value of DENOM extracted from the PGS calculator?
and additionally, I noticed the DENOM value is same for all individual accross all considered PGS IDs.

**Reply by smlmbrt** [reply]
From: https://pgsc-calc.readthedocs.io/en/latest/explanation/output.html

> `DENOM`: the number of non-missing genotypes used to calculate the PGS for this individual. 
> `AVG`: normalizes `SUM` by the `DENOM` field (displayed when you calculate the PGS on a small sample size n<50 to avoid using unreliable allele frequency estimates for missing genotypes in the target sample

Hopefully that answers, let me know if you have other questions!

**Reply by Djeane89** [reply]
@smlmbrt Thanks for your quick response :)

I did read the page. I am just curious whether I can obtain or see how the DENOM value is extracted from the outputs of the PGS calculator. I checked the output file (after executing the PGS), and I can only find the total number of matched variants from the considered target genotypes that were used in the score calculation.

Additionally, I thought that the DENOM would be different for each scoring file I used. For example, if I used three scoring files, then I would have three different values for DENOM, like this:

PGS0001.txt.gz --> DENOM X
PGS0002.txt.gz --> DENOM Y
PGS0003.txt.gz --> DENOM Z

Could you help clarify this just in case I misunderstood?
Looking forward to your response. Thank you

**Reply by smlmbrt** [reply]
Happy to explain! Basically, the scorefiles are combined into a single file (if the variant doesn't appear in a score it gets a weight of 0). This means that when we could the DENOM it will be the same for each score. If you were to run each of the scores separately they would have different DENOM values. The scores should be identically correlated.

**Reply by Djeane89** [reply]
@smlmbrt thanks alot for your explanation. It means alot :)

**Reply by Djeane89** [reply]
@smlmbrt oh I forgot to ask hehehe
Let's say I calculated the PRS from multiple scoring files, or separately, should I consider the 'AVG' column as the final PRS or the 'SUM'? Which one would you suggest?

**Reply by smlmbrt** [reply]
I'm assuming you didn't use the reference panel? If you didn't I would use the AVG and then probably Z-score it to make the distribution seem sensible.

**Reply by Djeane89** [reply]
@smlmbrt Yes, I didnt use it. Great!
Thank you :)
